src/main/java/io/codelex/arithmetic/practice/CheckOddEven.java

```diff
@@ -0,0 +1,15 @@
+package io.codelex.arithmetic.practice;
+
+public class CheckOddEven {
+    public static void main(String[] args){
+        Input(14);
+    }
+    public static void Input(int number){
```

Comment:
Metodes nosaukums ar lielo burtu, kā arī trūkst formatting - ieteiktu to settingos ieslēgt.